i18n/ads-language-pack-de/package.json

```diff
@@ -49,7 +49,7 @@
 						"path": "./translations/extensions/git-ui.i18n.json"
 					},
 					{
-						"id": "vscode.git",
+						"id": "Microsoft.git",
```

Comment:
The git extension is still published under the vscode publisher in our repo - is updating this correct?

Reply:
That is a small bug with the localization process, adding a small fix for that right now.